Formula/envoy.rb

```diff
@@ -65,11 +61,7 @@ def install
       --host_action_env=PATH=#{env_path}
     ]
 
-    if OS.linux?
-      # Disable extension `tcp_stats` which requires Linux headers >= 4.6
-      # It's a directive with absolute path `#include </usr/include/linux/tcp.h>`
-      args << "--//source/extensions/transport_sockets/tcp_stats:enabled=false"
```

Comment:
Should have sufficient Linux headers in Ubuntu 22. Absolute path was fixed in Envoy 1.22.0 (https://github.com/envoyproxy/envoy/commit/cac6264861c009f94f4c897c5210b84e9d614626) so source build should hopefully work with Homebrew `linux-headers@5.15` on older Linux that installs Homebrew `glibc`